hyperspy/io_plugins/hdf5.py

```diff
@@ -339,20 +339,40 @@ def dict2hdfgroup(dictionary, group, compression=None):
             group.attrs["_datetime_" + key] = repr(value)
         elif isinstance(value, list):
```

Comment:
It is possible, I was just lazy, as we have not decided we are going this way. So.. a PR with the example behaviour, but not quite ready for merge?